v3/src/components/axis/hooks/use-sub-axis.ts

```diff
@@ -93,9 +95,13 @@ export const useSubAxis = ({
         },
         renderNumericAxis = () => {
           select(subAxisElt).selectAll('*').remove()
-          const numericScale = d3Scale as unknown as ScaleLinear<number, number>,
-            axisScale = axis(numericScale).tickSizeOuter(0).tickFormat(format('.9')),
-            duration = isAnimating() ? transitionDuration : 0
+          const numericScale = d3Scale as unknown as ScaleLinear<number, number>
+          // When displaying bars, set the domain to [0, 100], otherwise use the default.
+          if (graphModel.pointDisplayType === "bars") {
```

Comment:
It didn't show up in the CodeCov report for some reason, but eslint reports `graphModel.pointDisplayType` as a missing dependency in this callback.